docs/resources/oxide_instance.md

```diff
@@ -8,50 +8,57 @@ This resource manages instances.
 
 -> Updates will stop and reboot the instance.
 
--> When setting a boot disk, the boot disk ID should also be included as part of `disk_attachments`.
```

Comment:
It's not entirely obvious that the boot disk must be part of disk attachments, and some people may not read all the way to the end. Let's leave this note please.

Reply:
Put the note back but modified it to use the word "must" since Terraform will error if the `boot_disk_id` is not present within `disk_attachments`.

```
Attribute "disk_attachments" must be specified when "boot_disk_id" is specified
```

Reply:
Excellent thanks!